makuai-corp-site/static/css/privacy.css

```diff
@@ -0,0 +1,83 @@
+/* セクションタイトル */
+.privacy-section1__title, .privacy-section2__title, .privacy-section3__title {
+    position: relative;
+    z-index: 0;
+    width: fit-content;
+    margin-left: 16px;
+    font: var(--main-title-font);
+    color: var(--main-title-color);
+}
+.privacy-section1__title::after, .privacy-section2__title::after, .privacy-section3__title::after {
+    position: absolute;
+    top: 46px;
+    left: -16px;
+    right: -16px;
+    z-index: -1;
+    height: 20px;
+    content: "";
+    background: var(--main-title-decoration);
+    opacity: 0.2;
+    border-radius: 3px;
+}
+
+/* テキスト */
+.privacy-section1__item, .privacy-section1__desc, .privacy-section2__item, .privacy-section2__desc, .privacy-section3__item, .privacy-section3__desc {
+    font: var(--main-desc-font);
+    color: var(--main-desc-color);
+    letter-spacing: 0.02em;
+}
+.privacy-section1__item, .privacy-section2__item, .privacy-section3__item {
+    font-weight: 700;
+    margin-bottom: 29px;
+}
+.privacy-section1__desc:not(:last-child), .privacy-section2__desc:not(:last-child), .privacy-section3__desc:not(:last-child) {
+    margin-bottom: 58px;
+}
+.privacy-section2__desc--emphasis {
+    color: var(--main-desc-color-emphasis);
+}
+
+/* セクション1 */
+.privacy-section1 {
+    max-width: 880px;
+    padding: 0 16px;
+    margin: 188px auto 120px;
```

Comment:
```suggestion
    padding: 188px 16px;
    margin: 0 auto 120px;
```
個人情報保護方針のページ(/privacy)でレイアウトが崩れています。
marginで余白を指定するとheaderの上にも余白ができてしまうので
paddingで指定するといいと思います。

<img width="1512" alt="スクリーンショット 2023-01-31 13 49 16" src="https://user-images.githubusercontent.com/103909058/215669052-0194e16b-b6d5-4bb3-9d73-2cfa8bf7bf74.png">